domains/madaha.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "catmaple",
+        "email": "szhaimei@hotmail.com"
+    },
+    "record": {
+        "CNAME": "www"
```

Comment:
invalid CNAME records. you can follow https://is-a.dev/docs for more information